ui/src/app/rsd/views/RSDList/RSDListHeader/RSDListHeader.test.tsx

```diff
@@ -82,17 +82,17 @@ describe("RSDListHeader", () => {
       pod: podStateFactory({ items: [podFactory()], loaded: true }),
     });
     const store = mockStore(state);
-    const setSelectedAction = jest.fn();
-    jest
-      .spyOn(React, "useState")
-      .mockImplementation(() => ["", setSelectedAction]);
-    mount(
+
+    const wrapper = mount(
       <Provider store={store}>
```

Comment:
Ended up just rewriting this failing test which expresses the same intent, but doesn't rely on a mock.